pypit/artrace.py

```diff
@@ -559,13 +601,11 @@ def trace_object(slf, det, sciframe, varframe, crmask, trim=2,
       Trim objects within xedge % of the slit edge
```

Comment:
A note here - I've just realised that `trim`, `triml`, `trimr`, and `xedge` are all doing the same thing (right?). My feeling is that we should have a standard that we use. My opinion is to use `triml` and `trimr` throughout this function, and if `triml` and `trimr` are not set, we can use either `trim` or `xedge` to calculate `triml` and `trimr`.

Reply:
I agree but I also forsee this method having a somewhat
significant overhaul.  So I am adding a TODO here but punting
elsewise.